Wangscape/noise/ModuleManager.h

```diff
@@ -14,14 +14,15 @@ namespace noise
 class ModuleManager
 {
 public:
+    using ReseedablePtr = noise::module::ReseedablePtr;
```

Comment:
Yes, I'm not sure why that's there. There's also `typedef noise::module::Module Module;`, which is totally unused in the class. I expect they're both relics from before the class was moved into the `noise` namespace.